backend/requirements_frozen.txt

```diff
@@ -1,17 +1,21 @@
 aiohttp==3.5.4
-async-lru==1.0.2
+aresponses==1.1.1
```

Comment:
I think we don't need this anymore, could you cleanup the dev requirements too?

Reply:
done